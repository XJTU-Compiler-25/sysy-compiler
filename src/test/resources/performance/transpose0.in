version https://git-lfs.github.com/spec/v1
oid sha256:2029ff2c91ce8309d74dbe7c08fff78dd3d5fcdc3cc641ee447dfc43daa9751e
size 102
