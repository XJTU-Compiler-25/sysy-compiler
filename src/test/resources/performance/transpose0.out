version https://git-lfs.github.com/spec/v1
oid sha256:e9e95c09e33ce8246dc4b47c7f33db12250043c718e4681b971fb5e999161d23
size 12
